Source/mhd/Castro_mhd.H

```diff
@@ -1,10 +1,50 @@
+
     void just_the_mhd(amrex::Real time, amrex::Real dt);
 
     void
     check_for_mhd_cfl_violation(const amrex::Box& bx, const amrex::Real dt,
                                 amrex::Array4<amrex::Real const> const& q_arr,
                                 amrex::Array4<amrex::Real const> const& qaux_arr);
 
+    void
+    plm(const amrex::Box& bx,
+        const int idir,
+        amrex::Array4<amrex::Real const> const& s,
+        amrex::Array4<amrex::Real const> const& qaux,
+        amrex::Array4<amrex::Real const> const& flatn,
+        amrex::Array4<amrex::Real const> const& Bx,
+        amrex::Array4<amrex::Real const> const& By,
+        amrex::Array4<amrex::Real const> const& Bz,
+        amrex::Array4<amrex::Real> const& qleft,
+        amrex::Array4<amrex::Real> const& qright,
+        amrex::Array4<amrex::Real const> const& srcQ,
+        const amrex::Real dt);
+
+    void
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
```

Comment:
These qualifiers need to go before the function type

Reply:
should be fixed now